src/main/webapp/app/entities/exercise.model.ts

```diff
@@ -123,6 +123,7 @@ export abstract class Exercise implements BaseEntity {
 
     // helper attributes
     public secondCorrectionEnabled = false;
+    public feedbackSuggestionsEnabled? = false;
```

Comment:
can this really be undefined?

Reply:
No 🙈 I think this might have been from an earlier draft of the PR. I changed it.

Reply:
Edit: It cannot be undefined, but it is needed to avoid [a few dozen type errors in tests that don't properly initialize client exercises](https://bamboo.ase.in.tum.de/browse/ARTEMIS-TESTS5371-TSTEST-86/log). Considering that this "fix" is also used for lots of other attributes here and I don't want to completely refactor all tests, I think I will keep the question mark here.